### PR TITLE
setup template-oss and linted

### PR DESCRIPTION
**Pull request by fritzy**
template-oss and linting

### CODE REVIEW

**Comment by fritzy** [comment]
Will need to land PR #101 first

File: tap-snapshots/test/dir.js.test.cjs [hunk 1]
```diff
@@ -316,7 +316,7 @@ Object {
 exports[`test/dir.js TAP with prepare script > extract 1`] = `
 Object {
   "from": "file:test/fixtures/prepare-script",
-  "integrity": "sha512-+bTA3RvkXgLMM1PIzfcwXkiiSYzCBHT6Jrqo8yt3owZtMhOnBomH1Dc0949tVurdyPk7WcrKitvsRnXRPCD4sQ==",
+  "integrity": "sha512-shf/7QYgFII06kJbyyqj4u86uLuyJnD0xVGLm0XDkC6nuVU+GBHwQ9uogbLUQnBu0gSvcWYVnO1TyPxj+YQDdw==",
```
Comment (ruyadorno) [inline tap-snapshots/test/dir.js.test.cjs:319]: are these integrity changes intentional?

File: test/remote.js [hunk 2]
```diff
@@ -86,27 +84,29 @@ t.test('packument', t => {
...
     })
 })
 
 t.test('bad integrity', t => {
   const url = `${server}/abbrev.tgz`
+  // eslint-disable-next-line max-len
   const integrity = 'sha512-AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA=='
   const f = new RemoteFetcher(url, { cache, integrity })
-  //return t.rejects(f.packument(), {
+  // return t.rejects(f.packument(), {
```
Comment (ruyadorno) [inline test/remote.js:103]: can this line just be removed?

Comment (fritzy) [reply test/remote.js]: removed

**Review by ruyadorno — APPROVED** [review]
LGTM 👍

**Pull request merged**